src/templates/actionBar.jsx

```diff
@@ -136,185 +164,217 @@ class ActionBar extends React.PureComponent {
-                                                                title={iconGrid.title || 'Grid View'}
-                                                                type="grid"
-                                                            />
-                                                            )}
-
-                                                            {/* eslint-disable max-len */}
-                                                            {iconSearch && (
-                                                            <Icon
-                                                                className="action_bar--search_icon"
-                                                                color={isMobileSearchVisible ?
-                                                                    'highlight' :
-                                                                    null}
-                                                                id={iconSearch.id}
-                                                                onClick={this.onMobileSearchIconToggle}
-                                                                title={iconSearch.title || 'Search'}
-                                                                type="search"
-                                                            />
-                                                            )}
-                                                            {/* eslint-enable max-len */}
-
-                                                            {iconTable && (
-                                                            <Icon
-                                                                color={iconTable.selected ?
-                                                                    'highlight' :
-                                                                    null}
-                                                                id={iconTable.id}
-                                                                onClick={iconTable.onClick}
-                                                                title={iconTable.title || 'Table View'}
-                                                                type="list"
-                                                            />
-                                                            )}
+                                                    })}
+                                                </DropdownButton>
+                                            )}
 
-                                                            {jsx}
-                                                        </List.Item>
-                                                    );
-                                                })}
-                                            </List>
+                                            {list && _.isArray(list) && (
```

Comment:
Is it necessary to do `list && _.isArray(list)`?  Shouldn't `_.isArray(list)` be sufficient?